components/kubernetes/kind-cluster.yaml

```diff
@@ -2,6 +2,9 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
+  extraMounts:
+  - hostPath: /proc
+    containerPath: /host/proc
```

Comment:
It's not clear why CWS needs _real_ `/host/proc` and how you'll be able to use that in a manifest that is not Kind-aware.